<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>storageDialog</class>
 <widget class="QDialog" name="storageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>521</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/middle.jpg</pixmap>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>521</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tabWidgetPage1_5">
    <attribute name="title">
     <string>商品信息查询</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_9">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>471</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_9">
      <item row="0" column="0">
       <widget class="QLabel" name="Label_34">
        <property name="text">
         <string>商品条码</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="info_code"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="info_submit">
        <property name="text">
         <string>显示</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="Label_35">
        <property name="text">
         <string>商品名称</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="info_name">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Label_2">
        <property name="text">
         <string>单价</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="info_price">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="Label_3">
        <property name="text">
         <string>供货商</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="info_sup">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWidgetPage2_5">
    <attribute name="title">
     <string>商品信息更新</string>
    </attribute>
    <widget class="QPushButton" name="update_submit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>320</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确认更新</string>
     </property>
    </widget>
    <widget class="QPushButton" name="update_clear">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>320</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>清除</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_8">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>471</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="Label_30">
        <property name="text">
         <string>商品条码</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="update_code"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="update_request">
        <property name="text">
         <string>显示</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="Label_31">
        <property name="text">
         <string>商品名称</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="update_name"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Label">
        <property name="text">
         <string>单价</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="update_price"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>库存信息查询</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_11">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>491</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_13">
      <item row="0" column="0">
       <widget class="QLabel" name="Label_50">
        <property name="text">
         <string>商品条码</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="storage_code"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="storage_submit">
        <property name="text">
         <string>显示</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="Label_51">
        <property name="text">
         <string>商品名称</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="storage_name">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Label_54">
        <property name="text">
         <string>剩余数量</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="storage_qty">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>最后进货日期</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="storage_lastin">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>库存信息更新</string>
    </attribute>
    <widget class="QPushButton" name="update_storage_clear">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>330</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>清除</string>
     </property>
    </widget>
    <widget class="QPushButton" name="update_storage_submit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>330</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确认更新</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_12">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>451</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_14">
      <item row="0" column="0">
       <widget class="QLabel" name="Label_55">
        <property name="text">
         <string>商品条码</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="update_storage_code"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="update_storage_request">
        <property name="text">
         <string>显示</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="Label_56">
        <property name="text">
         <string>商品名称</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="update_storage_name"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Label_59">
        <property name="text">
         <string>剩余数量</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="update_storage_qty"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
